app/models/statement.rb

```diff
@@ -0,0 +1,32 @@
+# Statement
+#
+# * A schedule has many statement
+#   -  A cohort/contract can have many statement
+# * A statement can have many declaration
+# * Not all statements have declarations
+# * A statement is comprised of:
+#   - service fees
+#   - output fee (individual declarations/outputs, uplifts)
+#   - vat?
+# * Generated by us (DfE)
+# * A statement can be ‘frozen’ preventing providers from voiding declarations
+#   and initializing clawbacks for any future voids. Refer to mark_as_paid field
+#   and logic.
+class Statement < ApplicationRecord
+  belongs_to :cohort
+  belongs_to :lead_provider
+  has_many :statement_items
+
+  validates :cohort_id, presence: true
+  validates :lead_provider_id, presence: true
+
+  validates :month, numericality: { in: 1..12, only_integer: true }
```

Comment:
I would prefer to go with strings for `month` instead of integers, ("january" or "jan" etc).

Reply:
@pmanrubia 
We currently have a `name` field with text "January 2024". We wanted to split this up into `month` and `year` to better validate it.
Migration shouldn't be too hard with this.

Reply:
I'd prefer the month number here because it's easier to sort things.

Easy enough to convert to text with `Date::MONTHNAMES[1]` if we need to in the UI.

Reply:
I'm not averse to us using a Postgres enumb for them, that'll let us sort properly _and_ refer to them by name. 

```sql
create type month as enum ('January', 'February', ...);
```

[Apparently Rails supports this properly now too](https://blog.saeloun.com/2022/01/12/rails-7-adds-custom-enum-support-in-postgresql/).

Reply:
discussed in person: we'd like to stick to integers due to sorting being easier